internal/cluster/refresh_status_preprocessor.go

```diff
@@ -134,6 +134,11 @@ func newValidations(log logrus.FieldLogger, api host.API) []validation {
 			condition: v.isNtpServerConfigured,
 			formatter: v.printNtpServerConfigured,
 		},
+		{
+			id:        isOCSMinRequirementsSatisfied,
+			condition: v.isOCSMinRequirementsSatisfied,
```

Comment:
IMO let's move OCS-specific code out of the generic code.  Here we can have a method that checks minimum requirements for all operators.  Then that method will call isOCSMinRequirementsSatisfied() and any other for other operators.

@filanov What do you think?

Reply:
Sounds good to me